airbyte-integrations/connectors/source-clickhouse/src/main/java/io/airbyte/integrations/source/clickhouse/ClickHouseSource.java

```diff
@@ -90,9 +90,19 @@ public JsonNode toDatabaseConfig(final JsonNode config) {
         config.get(JdbcUtils.PORT_KEY).asText(),
         config.get(JdbcUtils.DATABASE_KEY).asText()));
 
+    boolean isAdditionalParamsExists = config.get(JdbcUtils.JDBC_URL_PARAMS_KEY) != null && !config.get(JdbcUtils.JDBC_URL_PARAMS_KEY).asText().isEmpty();
+    List<String> params = new ArrayList<>();
     // assume ssl if not explicitly mentioned.
     if (isSsl) {
-      jdbcUrl.append("?").append(SSL_MODE);
+      params.add(SSL_MODE);
+    }
+    if (isAdditionalParamsExists) {
+      params.add(config.get(JdbcUtils.JDBC_URL_PARAMS_KEY).asText());
```

Comment:
what will happen if the user puts "sslmode=strict" into `jdbc_url_params` field? Specifically, will the connection continue to work and what the effective SSL mode of the connection will be in this case?